packages/google_maps_flutter/google_maps_flutter/example/android/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.1'
+        classpath 'com.android.tools.build:gradle:7.3.0'
```

Comment:
And in general, why are you changing Android example configuration in this PR at all?